metadata_layer.py

```diff
@@ -1244,9 +1430,9 @@ def make_thumbnail(self, pool, data_source, link, link_obj):
         # The thumbnail and image are different. Make sure there's a
         # separate link to the thumbnail.
         thumbnail_obj, ignore = link_obj.identifier.add_link(
-            rel=thumbnail.rel, href=thumbnail.href,
-            data_source=data_source,
-            license_pool=pool, media_type=thumbnail.media_type,
+            rel=thumbnail.rel, href=thumbnail.href, 
+            data_source=data_source, 
+            license_pool=None, media_type=thumbnail.media_type,
```

Comment:
In the future branch, the link won't have a pool anymore. But until that happens, we should make sure it has the right pool in case something's using it.